modules/administration-guide/partials/proc_using-chectl-to-install-che-on-codeready-containers.adoc

```diff
@@ -15,21 +15,21 @@ This section describes how to install {prod-short} on CodeReady Containers (CRC)
 
 .Procedure
 
-. Initiate the CRC platform:
-
-.. Configure your host machine for CodeReady Containers:
+.. Remove any previous cluster:
 +
 ----
-$ crc setup
+$ crc delete
 ----
 
-.. Remove any previous cluster:
+. Initiate the CodeReady Containers platform:
+
+.. Configure your host machine for CodeReady Containers:
 +
 ----
-$ crc delete
+$ crc setup
 ----
 
-.. Initiate the `oc` command line interface within CRC: 
+.. Enable access to the `oc` command line interface embedded in CodeReady Containers:
```

Comment:
Should happen *after* crc start, or you receive the error:

```
eval $(crc oc-env)
Machine does not exist. Use 'crc start' to create it
```

Reply:
Suggestions in https://github.com/ibuziuk/che-docs/pull/3

*    Use CodeReady containers rather than CRC
*    Fix steps numbering.
*  Tested the procedure: removing obsolete commands, reordering commands.

Reply:
@themr0c applied, thanks